alcs-api/docker-compose.yml

```diff
@@ -56,6 +54,20 @@ services:
     networks:
       - alcs
 
+#  metabase:
```

Comment:
I figure if you want to run metabase locally you can uncomment it, but don't need metabase running all the time. It will also create tables in same schema as postgres.